python/tvm/relay/frontend/tensorflow.py

```diff
@@ -835,6 +847,15 @@ def _impl(inputs, attr, params):
         return AttrCvt(op_name="where")(inputs, attr)
     return _impl
 
+def _reverse_v2():
+    def _impl(inputs, attr, params):
+        axis = params.pop(inputs[1].name_hint).asnumpy()[0]
+        return AttrCvt(
+            op_name="reverse",
+            ignores=['Tidx'],
+            extras={'axis': axis})([inputs[0]], attr)
```

Comment:
axis should be int(axis) to fix CI error

Reply:
@lixiaoquan fixed, thanks